NebulaWorld/UIPlayerList/UIStyles/DialogStyles.cs

```diff
@@ -0,0 +1,42 @@
+﻿using UnityEngine;
+
+namespace NebulaWorld.UIPlayerList.UIStyles
+{
+    internal class DialogStyles
+    {
+        // ReSharper disable once InconsistentNaming
+        private static Texture2D _textureMemory = null;
+
+        public static GUIStyle WindowBackgroundStyle()
+        {
+            var width = 32;
+            var height = 32;
+
+            if (_textureMemory == null)
+                _textureMemory = MakeTex(width, height, new Color32(36, 67, 76, 200);
```

Comment:
Error	CS1026	) expected

Reply:
I sure am curious to how i even built this with that missing... but It's fixed now.